src/components/hierarchy-bar/hierarchy-bar-node/hierarchy-bar-node.component.html

```diff
@@ -28,14 +38,55 @@
             aria-label="Show children"
             role="button"
             class="hierarchy-bar-node-arrow"
+            [class.hierarchy-bar-node-arrow-readonly]="mode === 'readOnly'"
             [uxPopover]="popoverTemplate"
             [popoverContext]="{ node: node, popover: popover }"
             placement="bottom"
             popoverClass="hierarchy-bar-popover"
-            [showTriggers]="hierarchyBar.popoverShowTriggers"
-            [hideTriggers]="hierarchyBar.popoverHideTriggers"
+            [showTriggers]="mode !== 'readOnly' ? hierarchyBar.popoverShowTriggers : null"
+            [hideTriggers]="mode !== 'readOnly' ? hierarchyBar.popoverHideTriggers : null"
             tabindex="0">
-        <ux-icon name="next" class="hierarchy-bar-node-arrow-icon"></ux-icon>
+        <ux-icon name="next"
+                 class="hierarchy-bar-node-arrow-icon">
+        </ux-icon>
+    </button>
+</ng-template>
+
+<ng-template #alternativeClickMode>
+    <button type="button"
+            uxFocusIndicator
+            uxFocusIndicatorOrigin
+            #popover="ux-popover"
+            aria-label="Show children"
+            class="hierarchy-bar-node-content"
+            [uxPopover]="popoverTemplate"
+            [popoverContext]="{ node: node, popover: popover }"
+            placement="bottom"
+            popoverClass="hierarchy-bar-popover"
+            [showTriggers]="node.children ? hierarchyBar.popoverShowTriggers : null"
+            [hideTriggers]="node.children ? hierarchyBar.popoverHideTriggers : null"
+            tabindex="0"
+            [attr.aria-label]="node.title"
+            (click)="node.children ? selected.emit(node) : null">
+
+        <!-- Show a custom icon if specified -->
+        <div class="hierarchy-bar-node-icon" *ngIf="hierarchyBar.icon">
+            <ng-container [ngTemplateOutlet]="hierarchyBar.icon" [ngTemplateOutletContext]="{ node: node, $implicit: node }"></ng-container>
+        </div>
+
+        <!-- Show an icon if specified -->
+        <img class="hierarchy-bar-node-icon" *ngIf="node.icon && !hierarchyBar.icon" [src]="node.icon" alt="Hierarchy Bar Icon">
+
+        <!-- Show the name of the current node -->
+        <span class="hierarchy-bar-node-title">{{ node.title }}</span>
+
+        <!-- Show a dropdown arrow if there are children -->
+        <div class="hierarchy-bar-node-arrow hierarchy-bar-node-arrow-icon-alternative">
```

Comment:
If the parent button is hovered, this icon should have the same primary color highlight as seen in the "standard" mode.